client/my-sites/site-settings/start-over.jsx

```diff
@@ -113,7 +118,50 @@ function SiteResetCard( {
 	const dispatch = useDispatch();
 
 	const { data } = useSiteResetContentSummaryQuery( siteId );
+	const {
+		data: status,
+		refetch: refetchResetStatus,
+		isFetching: isFetchingResetStatus,
+	} = useSiteResetStatusQuery( siteId );
+	const [ resetStatus, setResetStatus ] = useState( status?.status ?? 'completed' );
+	const [ wasResetting, setWasReseting ] = useState( false );
 	const [ isDomainConfirmed, setDomainConfirmed ] = useState( false );
+	const [ resetProgress, setResetProgress ] = useState( 0.1 );
+
+	useEffect( () => {
+		let interval = 1;
+		if ( ! siteId || resetStatus === 'completed' ) {
+			return;
+		}
+		if ( ! isFetchingResetStatus ) {
+			interval = setInterval( async () => {
+				const {
+					data: { status: latestStatus },
+				} = await refetchResetStatus();
+				setResetStatus( latestStatus );
+			}, 3000 );
+		}
+		return () => clearInterval( interval );
+	}, [ isFetchingResetStatus, refetchResetStatus, resetStatus, siteId ] );
```

Comment:
Maybe we can use the `Tick` component instead of implementing polling manually 👍